gadget/gadget_test.go

```diff
@@ -1586,23 +1586,19 @@ volumes:
 	err := ioutil.WriteFile(s.gadgetYamlPath, []byte(gadgetYaml), 0644)
 	c.Assert(err, IsNil)
 
-	_, err = gadget.ReadInfo(s.dir, nil)
+	_, err = gadget.ReadInfo(s.dir, nil, nil)
 	c.Check(err, ErrorMatches, `invalid volume "pc": structure #1 \("mbr"\) has "mbr" role and must start at offset 0`)
 }
 
-func (s *gadgetYamlTestSuite) TestGadgetConsistencyWithoutConstraints(c *C) {
+func (s *gadgetYamlTestSuite) TestReaindInfoValidateConsistencyWithoutConstraints(c *C) {
```

Comment:
```suggestion
func (s *gadgetYamlTestSuite) TestReadInfoValidatesConsistencyWithoutConstraints(c *C) {
```

or at least that's what I think the name should be? :-)